qiskit_alice_bob_provider/plugins/state_preparation.py

```diff
@@ -49,16 +45,40 @@ def _reset_prep() -> Instruction:
     return Reset()
 
 
-def _get_unitary_synthesis(config: PassManagerConfig):
-    return CustomUnitarySynthesis(
-        config.basis_gates,
-        approximation_degree=config.approximation_degree,
-        coupling_map=config.coupling_map,
-        backend_props=config.backend_properties,
-        plugin_config=config.unitary_synthesis_plugin_config,
-        method=config.unitary_synthesis_method,
-        target=config.target,
-    )
+class CustomSolovayKitaev(TransformationPass):
```

Comment:
Is this class a custom version of https://github.com/Qiskit/qiskit/blob/stable/1.4/qiskit/transpiler/passes/synthesis/solovay_kitaev_synthesis.py#L41-L189 ?

If yes, I'm thinking it might be worth it to add a good explanation in the docstring, to explain the differences with the qiskit version (in implementation and in behavior, maybe with a couple examples?), so that in the future it's easier to understand why these (tricky) changes were necessary 😬